### PR TITLE
V2 - Disable Services

### PR DESCRIPTION
**Pull request by marcus-snx**
- remove postgres database (and foreign migration for parquet scripts)
- replace indexers service with new version
- remove the postgres and individual processor services from `docker-compose.yaml`

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -215,13 +43,10 @@ services:
     restart: always
     networks:
       - data
-    depends_on:
-      - db
     env_file:
       - .env
     environment:
       AIRFLOW__CORE__EXECUTOR: LocalExecutor
-      AIRFLOW__DATABASE__SQL_ALCHEMY_CONN: postgresql+psycopg2://postgres:$PG_PASSWORD@db:5432/postgres
```
Comment (Tburm) [inline docker-compose.yml:224]: Can we actually do this part without impacting airflow? I thought this refers to airflow using the database to store information about jobs. It's possible this isn't actually happening so everything will be fine though.

File: docker-compose.yml [hunk 2]
```diff
@@ -1,37 +1,7 @@
...
-    ports:
-      - "${DB_PORT}:5432"
-    deploy:
-      resources:
-        limits:
-          cpus: "4.0"
-          memory: 8192M
-    shm_size: 4gb
-    tmpfs:
-      - /dev/shm:size=4g
```
Comment (Tburm) [inline docker-compose.yml:30]: Need to check if this impacts airflow. If it does, I think we can scale it down significantly but may need to keep some of this.

**Comment by marcus-snx** [comment]
@Tburm I've added the postgres service again, will test it out and see how much we can downscale it once I have the pipeline setup done for the scheduler part

**Comment by Tburm** [comment]
image
this is my favorite part

**Pull request merged**